CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### [3.22.0] 2022-09-28
+
+### Changed
+
+- The package now exposes a `__version__` attribute.
+- The contraint of the `pyarrow` dependency has been loosened from `<7` to `<9`.
```

Comment:
Do we need to explain we bump snowflake along the way too ? 🤔

Reply:
It's a patch release bump, so I'd say no